src/commands.h

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Not required for this commit but good practice to include #pragma once in g++ header files.

Reply:
Yes forgot to add, will fix in next PR